tools/mfassignr/mfassignr_isofiltr.xml

```diff
@@ -0,0 +1,43 @@
+<tool id="IsoFiltR" name="IsoFiltR: Isotope Filtering" version="@TOOL_VERSION@+galaxy0">
```

Comment:
I would consider having 'mfassignr_isofiltr' for the `tool id` and 'MFAssignR IsoFiltR' for the `name` to specify it comes from the MFAssignR package :)

Reply:
done in 6b41efd